includes/functions_install.php

```diff
@@ -47,7 +47,7 @@ class module
 
 function get_db_tools($db)
 {
-	if (defined('PHPBB_32'))
+	if (defined('PHPBB_32') || defined('PHPBB_33'))
```

Comment:
I think we should just check if this factory exists instead of trying it to versions. Otherwise we're going to be doing this every new version and I'd rather we not.

Reply:
It does exists in 3.3.o indeed.

Reply:
Right, so I'd rather we check if the class exists. Maybe:
```
if (class_exists('\phpbb\db\tools\factory'))
```

Tying functionality or checks to versions should really be avoided.